docs/source/operations/baf_verify.md

```diff
@@ -237,3 +237,56 @@ For final checking of the validity of the indy network.
+| Q1      | g           | **Ansible playbook failed**  <br> Playbook execution terminated at <br> **Role**: create/crypto/ibft <br> **Task**: Copy the crypto material to Vault <br> **Error**: Vault Access denied, Root Token invalid, Vault Sealed                                                                                                                                            | Issue with Vault connectivity                                                                                                          | If the logs mention "access denied", make sure that the Vault authentications were created correctly by checking all the tabs on Vault UI. <br> Any Vault authentication problem is because of running different configurations (network.yaml) on the same Vault. Please ensure that you reset the network before re-running with a different network.yaml.                                                                                                                                                                                                                                                                                                                              |
+| Q1      | h           | **Ansible playbook failed**  <br> Playbook execution terminated at <br> **Role**: create/tessera or create/constellation <br> **Task**: Create value file for Tessera TM for each node <br>  **Error**: could not locate file in lookup: network.config.genesis                                                                                                                                            | Genesis file not present in the location/not added in configuration file                                                                                                           | It happens mainly when adding new organization to the exising network. There are two ways to keep the genesis file, one is to give the genesis file path of the exising network or hardcode the base64 encoded genesis in the configuration file.                                                                                                                                                                                                                                                                                 |

```

Comment:
Can we give the genesis file path as well?
I thought we need to hardcode the genesis file content into the network.yaml

Reply:
yes. we can give the genesis file path as well.